syft/federated/plan.py

```diff
@@ -509,3 +512,56 @@ def __str__(self):
         out += ">"
 
         return out
+
+
+    @staticmethod
+    def _simplify_plan(plan: sy.Plan) -> tuple:
```

Comment:
you can call it just simply and detail in all objects now.

Reply:
Furthermore, you can just use Plan rather than sy.plan since it is inside the class itself

Reply:
Sorry the imports I made are not correct. It looks like I can't use the object Plan as a parameter of the method within the Plan class.